network/servicenetwork/servicenetwork_test.go

```diff
@@ -353,3 +362,22 @@ func TestServiceNetwork_SendCascadeMessage2(t *testing.T) {
 	require.True(t, success)
 }
 */
+
+func TestEcdsaMarshalUnmarshal(t *testing.T) {
```

Comment:
Move this test to platformpolicy package.

Reply:
done